### ISSUE TITLE
Move collapsed Breadcrumb icon to separate tag

### ISSUE THREAD

**Issue by salarenko** [SAP/fundamental-styles#1735]
**Describe the bug**
Breadcrumb expand collapsed icon should be moved to separate `<i>` tag and used together with `sap-icon` class.

**Expected behavior**
#### NOW
```
<span class="fd-breadcrumb__dropdown-icon"></span>
```

#### PROPOSED
```
<i aria-label="Show collapsed breadcrumbs" class="fd-breadcrumb__dropdown-icon sap-icon--slim-arrow-down"></i>
```

**Comment by InnaAtanasova** [comment]
0.5day
